src/python/tests/core/bot/fuzzers/libFuzzer/libfuzzer_strategy_selection_test.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for strategy selection file."""
+
+import unittest
+
+from bot.fuzzers import strategy
+from bot.fuzzers.libFuzzer import strategy_selection
+from tests.test_libs import helpers as test_helpers
+
+
+class TestStrategySelectionGeneratorPatch(unittest.TestCase):
+  """Tests whether program properly generates strategy
+  sets for use by the launcher."""
+
+  def setUp(self):
+    """Set up method for strategy pool generator tests with patch."""
+    test_helpers.patch_environ(self)
+    test_helpers.patch(self,
+                       ['bot.fuzzers.engine_common.decide_with_probability'])
+    self.mock.decide_with_probability.return_value = True
+
+  def test_random_pool_generator(self):
+    """Deterministically tests the random strategy pool generator."""
+    strategy_pool = strategy_selection.generate_strategy_pool()
+    self.assertTrue(strategy_pool[strategy.CORPUS_MUTATION_RADAMSA_STRATEGY])
+    self.assertFalse(strategy_pool[strategy.CORPUS_MUTATION_ML_RNN_STRATEGY])
```

Comment:
Could you explain these first two with a comment? Since this depends on the implementation, it would be good to document why we expect ml rnn to be False instead of radamsa in case someone updates the code later and wants to know if changing the order actually broke something or if they just need to update the test.

Reply:
Sure, but not sure I understand. Changing the order of what?

Reply:
Discussed offline, the current wording LGTM.